docs/notebooks/Data.ipynb

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
This is awesome!
Might be nice to rename it to something a bit more descriptive - DataQueries? or something

Reply:
Update name